packages/viewer/src/types/events.ts

```diff
@@ -0,0 +1,18 @@
+export interface EventConfig {
+  /**
+   * The number of milliseconds after a single tap where a second tap will
+   * trigger the "doubletap" event.
+   */
+  doubleTapThreshold: number;
+
+  /**
+   * The number of milliseconds that a tap or click has to be held down
+   * before triggering the "longpress" event.
+   */
+  longPressThreshold: number;
+}
+
+export const defaultEventConfig: EventConfig = {
+  doubleTapThreshold: 500,
```

Comment:
Can't remember, do we have double click in the connect app? Is this what we use there? The value seems a bit high.

Reply:
Yep, we do have a double click gesture in Connect. That's also set to 500ms, but I agree that it might be a bit high. I think that threshold was set quite some time ago and just never revisited. Certainly open to adjusting these